web-app/js/smartR/_angular/services/controlElements.js

```diff
@@ -0,0 +1,373 @@
+//# sourceURL=controlElements.js
```

Comment:
It's sadly not so easy as the buttons are heavily linked to the code they are in. For now I moved them into a service to make jshint happy and to get it out of smartR.js.
I will probably make them directives once I overhaul the control elements in all visualisations, but for now this will work.